src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -368,45 +370,44 @@ const createMetacard = async (parent, args, context) => {
 }
 
 const saveMetacard = async (parent, args, context) => {
-  const { id, attrs } = args
+  const { id, attributes } = args
   if (
-    Array.isArray(attrs.metacard_tags) &&
-    attrs.metacard_tags.includes('query-template')
+    Array.isArray(attributes.metacard_tags) &&
+    attributes.metacard_tags.includes('query-template')
   ) {
     return await saveQueryTemplate(parent, args, context)
   }
 
-  const { fetch, fromGraphqlName, toGraphqlName } = context
+  const [oldMetacard] = await metacardsById(
+    parent,
+    { ids: [id], ...args },
+    context
+  )
+  const [oldMetacardAttrs] = oldMetacard.attributes
+  const { catalog, fromGraphqlName, toGraphqlName } = context
 
-  const attributes = Object.keys(attrs).map(attribute => {
-    const value = attrs[attribute]
-    return {
-      attribute: fromGraphqlName(attribute),
-      values: Array.isArray(value) ? value : [value],
+  Object.keys(oldMetacardAttrs).forEach(attrs => {
+    if (attributes[attrs] === undefined) {
+      attributes[attrs] = oldMetacardAttrs[attrs]
     }
   })
+  const body = {
+    metacards: [
+      {
+        ids: [id],
+        metacardType: args.attributes.metacard_type,
+        attributes,
+      },
+    ],
+  }
+  const res = await catalog.update(body)
```

Comment:
`renameKeys` here https://github.com/connexta/revelio/pull/184/files#diff-fd0e9d005f52564b980b4cf83cbd2fe3R246 
is causing issues when trying to apply update. need to address